src/components/ens-registration/SearchResult/SearchResultGradientIndicator.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import LinearGradient from 'react-native-linear-gradient';
+import styled from 'styled-components';
+import { Text } from '../../text';
+import { useTheme } from '@rainbow-me/context';
+import { useDimensions } from '@rainbow-me/hooks';
+
+type Props = {
+  type: 'availability' | 'expiration' | 'price';
+  isRegistered?: boolean;
+  price?: string;
+  expiryDate?: string;
+};
+
+const GradientBackground = styled(LinearGradient).attrs(({ colors }) => ({
```

Comment:
Ideally, with the design system, we shouldn't be using styled-components. Can we convert this to use the design system? I'm assuming it would look something like:

```tsx
<Box 
  as={LinearGradient} 
  colors={colors} 
  end={{ x: 1, y: 0}} 
  start={{ x: 0, y: 1}} 
  alignItems="center"
  justifyContent="center"
  height="40px"
>
  <Inset horizontal="15px">
    <Text
        color={textColor}
        containsEmoji={type === 'availability'}
        size={isSmallPhone ? '18px' : '20px'}
        weight="heavy"
      >
        {text}
      </Text>
  </Inset>
</Box>
```

Reply:
yes that looks great, design system advice is much appreciated!

Reply:
:D